src/Esfa.Vacancy.Register.Api/App_Start/ApiFilterConfig.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Web.Http;
+using System.Web.Http.Filters;
+
+namespace Esfa.Vacancy.Register.Api
+{
+    public class ApiFilterConfig
+    {
+        public void RegisterFilters(HttpFilterCollection filters)
+        {
+            filters.Add(new AuthorizeAttribute());
```

Comment:
Could this not be a static method similar to how the other AppStart classes bootstrap.

Reply:
Sure, no other dependencies in WebApiConfig are injectable so why not.